### PR TITLE
Moved/renamed license

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by aashish24** [comment]
@williams13 @charlesdoutriaux should we use the Apache V2 license instead of using the one we have?

**Comment by aashish24** [comment]
@danlipsa please review

**Comment by doutriaux1** [comment]
@aashish24 not sure we can, I think the lab requires this one. What is the advantage of Apache2?

**Comment by chaosphere2112** [comment]
http://www.apache.org/licenses/

**Comment by williams13** [comment]
Let’s discuss this next week.

-Dean

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,5 @@
-CCLRC Licence for CCLRC Software forming part of the 
- Climate Data Analysis Tools Package
+CCLRC License for CCLRC Software forming part of the  Ultrascale Visualization and
```
Comment (doutriaux1) [inline LICENSE]: I don't think there's a `and` here @williams13 ?

**Comment by aashish24** [comment]
@doutriaux1 please have a look here: http://choosealicense.com/

The apache license is nice since it is well defined and just a lit bit restrictive than bsd for good reasons. Also, it is compatible with GPL3 (but not GPL2). It takes care of patents issues (not sure if this will affect us). NASA encourages Apache.

**Comment by durack1** [comment]
@aashish24 @doutriaux1 @chaosphere2112 @williams13 I'd note there is an open issue about this #1354 - would be best to link these so as the issue is resolved it's closed.

**Comment by aashish24** [comment]
Folks  I am merging this one in once we discuss the issue more in detail at AGU or past AGU, we can update the branch. Thanks to @durack1 for referring to the existing issue.

**Pull request merged**